src/components/Form.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import './styles/Form.css';
+
+const Form = () => {
+  const formSubmitHandler = (event) => {
+    event.preventDefault();
+  };
+  return (
+    <section className="form">
+      <form>
+        <h2 className="new-book">ADD NEW BOOK</h2>
+        <div className="form-inputs">
+          <input
+            type="text"
+            className="text"
+            name="title"
+            placeholder="Title"
+            required
+          />
+          <select
+            name="category"
+            className="category-select"
+            defaultValue=""
+            required
+          >
+            <option value="" hidden>
+              Category
+            </option>
+            <option value="business">Business</option>
+            <option value="fiction">Fiction</option>
+            <option value="programming">Programming</option>
+          </select>
+          <button type="submit" onClick={formSubmitHandler}>
+            ADD BOOK
+          </button>
+        </div>
+      </form>
```

Comment:
- [OPTIONAL] I believe Teddy miss these out in his first review. Please add an author input here. Even though the template design doesn't have an author field but it will make more sense to have it in our app. Because in reality, a book only has an author.